### PR TITLE
feat(MustConn): Add MustConnectionString on (some) dbs

### PR DESCRIPTION
**Pull request by bearrito**
## What does this PR do?

Adds `MustConnectionString` on a variety of DB containers. I skipped MSSQL because that was failing to run tests for me locally.

There are probably more I can add them. Should I just add them to all containers with a `ConnectionString` method?

## Why is it important?

Not sure again, just looking at `good first issue` tags. 
https://github.com/testcontainers/testcontainers-go/issues/2134

## How to test this PR
Run the unit tests.

One point about the unit tests, I could not figure out a way to get a `panic` out of the `ConnectionString` methods. I looked for ways to corrupt/mutate the exposed_ports and host, but could not figure out an entrypoint. So there is no test that it actually panics if unable to provide the connstring.

### CODE REVIEW

File: modules/mysql/mysql_test.go [hunk 1]
```diff
@@ -34,6 +35,8 @@ func TestMySQL(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	mustConnectionString := container.MustConnectionString(ctx,"tls=skip-verify")
+	require.Equal(t,connectionString,mustConnectionString)
```
Comment (mdelapenya) [inline modules/mysql/mysql_test.go:39]: Super NIT: all the assertions in this module are done without testify.

I have to think about if using testify everywhere, or remove it everywhere, but not there yet. I'd keep this file at it was, without testify 🙏

File: modules/nats/nats_test.go [hunk 2]
```diff
@@ -32,6 +33,8 @@ func TestNATS(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to get connection string: %s", err)
 	}
+	mustUri := container.MustConnectionString(ctx)
+	require.Equal(t,uri,mustUri)
```
Comment (mdelapenya) [inline modules/nats/nats_test.go:37]: Same here about testify.

**Review by mdelapenya — COMMENTED** [review]
> One point about the unit tests, I could not figure out a way to get a panic out of the ConnectionString methods.

It's fine, I would not worry to get 100% coverage meanwhile it's clear, which this PR is.

Other than that, LGTM, thank you!

File: modules/mysql/mysql_test.go (hunk 1 above)
Comment (bearrito) [reply modules/mysql/mysql_test.go:39]: Will do.

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks for implementing this :clap:

**Pull request merged**